### PR TITLE
fix: set secondary color to editprofile buttons 

### PR DESCRIPTION
**Pull request by addiegupta**
Fixes: #1532

Changes:
- Changed color of update button and profile photo FAB to secondary color by using app theme
- Refactored 'buttonUpdate' to 'updateButton' for consistency
- Moved a hardcoded string to string resources

Screenshots for the change:

new button pr non rounded

### CODE REVIEW

File: app/src/main/res/layout/fragment_edit_profile.xml [hunk 1]
```diff
@@ -79,12 +78,11 @@
 
         </LinearLayout>
 
-        <androidx.appcompat.widget.AppCompatButton
-            android:id="@+id/buttonUpdate"
+        <com.google.android.material.button.MaterialButton
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_edit_profile.xml]: Just use Button

Should work

Comment (addiegupta) [reply app/src/main/res/layout/fragment_edit_profile.xml]: Are you sure? MaterialButton has different ripples and looks better IMO

Comment (iamareebjamal) [reply app/src/main/res/layout/fragment_edit_profile.xml]: Is there a difference, the library should automatically inflate Button to MaterialButton

Comment (addiegupta) [reply app/src/main/res/layout/fragment_edit_profile.xml]: Changed MaterialButton to Button in latest push

**Comment by iamareebjamal** [comment]
Is it working as expected?

**Comment by addiegupta** [comment]
> Is it working as expected?

yes

**Pull request merged**